gallery/visualtests/nxTag.js

```diff
@@ -46,17 +49,22 @@ describe('NxTag', function() {
   });
 
   describe('NxSelectableTag', function() {
-    const selector = '#nx-selectable-tag-example .nx-tag:first-child';
+    const selector = '#nx-selectable-tag-example .nx-tag:first-child',
+        lastTagSelector = '#nx-selectable-tag-example .nx-tag:last-child';
 
-    it('has a blue/grey border and lighter blue/grey background by default', simpleTest(selector));
+    it('has a lighter background and darker border by default', simpleTest(selector));
     it('has a dark grey border when hovered', hoverTest(selector));
-    it('has a light blue glow and light blue border when focused', focusTest(selector));
-    it('has a light blue glow and blue border when focused and hovered', focusAndHoverTest(selector));
+    it('has a blue outline when focused', focusTest(selector, undefined, OUTSET));
+    it('has a blue outline and dark grey border when focused and hovered',
+        focusAndHoverTest(selector, undefined, OUTSET));
 
-    it('has a dark blue/grey background when clicked', async function() {
-      const [targetElement] = await waitAndGetElements(selector);
+    it('has a darker background and no border when selected', async function() {
+      const [targetElement, lastElement] = await waitAndGetElements(selector, lastTagSelector);
 
       await targetElement.click();
+      //remove focus from tag for snapshot
```

Comment:
I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!

Reply:
> I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!

NxSelectableTag is built around a checkbox input.  That's what you'd need to blur

Reply:
> > I couldn't figure out how to blur the element here - I tried blurring NxTag and the icon inside, but both left the focus outline in the snapshot. Any other ideas to remove the border are appreciated!
> 
> NxSelectableTag is built around a checkbox input. That's what you'd need to blur

Ah of course, that makes sense. Thanks for the suggestion/feedback!